packages/bot/src/components/transactions.jsx

```diff
@@ -0,0 +1,169 @@
+                                    alignment='left'
+                                    message={translate('Buy price')}
+                                >
+                                    <BuyPriceIcon className='transactions__middle' />
+                                </Popover>
+
+                                <Money
+                                    amount={Math.abs(contract.buy_price)}
+                                    currency={contract.currency}
+                                />
+                            </div>
+                        </div>
+                    </td>
+                    <td className='transactions__middle transactions__td'>
+                        <div>
+                            <Popover
+                                className='transactions__inline transactions__middle'
+                                alignment='left'
+                                message={translate('Entry spot')}
+                            >
+                                <EntrySpotIcon className='transactions__middle' />
+                            </Popover>
+                            <div className='transactions__inline transactions__middle'>
+                                {contract.entry_spot}
+                            </div>
+                        </div>
+                        <div>
+                            <Popover
+                                className='transactions__inline transactions__middle'
+                                alignment='left'
+                                message={translate('Exit spot')}
+                            >
+                                <ExitSpotIcon className='transactions__middle' />
+                            </Popover>
+                            <div className='transactions__inline transactions__middle'>
+                                {contract.exit_spot}
+                            </div>
+                        </div>
+                    </td>
+                    <td className='transactions__td'>
+                        <div
+                            className='transactions__inline
+                        transactions__middle
+                        transactions__green'
+                        >
+                            <Money
+                                amount={Math.abs(contract.profit)}
+                                currency={contract.currency}
+                            />
+                        </div>
+                       
+                    </td>
+                    <td className='transactions__td'>
+                        {
+                            contract.is_settled ?
+                                <Popover
+                                    className='transactions__inline transactions__middle'
+                                    alignment='left'
+                                    message={translate('Settled')}
```

Comment:
Since we're not _settling_ the contract anymore, could this be renamed?

```suggestion
                                    message={translate('Completed')}
```